tasmota/tasmota_template.h

```diff
@@ -152,6 +152,9 @@ enum UserSelectablePins {
   GPIO_TM1637CLK, GPIO_TM1637DIO,      // TM1637 interface
   GPIO_PROJECTOR_CTRL_TX, GPIO_PROJECTOR_CTRL_RX,  // LCD/DLP Projector Serial Control
   GPIO_SSD1351_DC,
+#ifdef USE_XPT2046
+  GPIO_XPT2046_CS,                     // XPT2046 SPI Chip Select
+#endif
   GPIO_SENSOR_END };
```

Comment:
Do not use ``#ifdef USE_XPT2046`` here as it needs to be compiled in always to keep the list complete.

So pls remove these (only here).

Reply:
Done, ifdefs removed and committed